src/types/types.tsx

```diff
@@ -6,27 +8,40 @@ export type Vendor = {
   name: string;
 };
 
+export type VoucherRange = {
+  startSerialNum: number;
+  endSerialNum: number;
+  type: string;
```

Comment:
Consider using an enum for the type -- why was `VoucherType` removed?

Reply:
We decided to remove it because the VoucherType will never be entered in the app but only through the admin dashboard — in case the admins ever want to add a new VoucherType, we'll have to add another type in the enum but the string can just work as is.

Reply:
yeh this was a database change. instead of users manually entering types and us just checking if the type exists, what happens in admins will create a new set of valid vouchers - say serial numbers 100-200 with color purple. when we validate a voucher's serial number, we perform a lookup on the voucher-ranges database to find the range it belongs to and automatically set the type. and like sauhard said, this has the added bonus of not needing to update any code if they want to add more types.

Reply:
Ah good thinking -- yes that makes sense to me! I'd just make a note of basically exactly what you just said in the PR description so it's documented clearly!